### ISSUE TITLE
Initializer member complete too aggressive at first

### ISSUE THREAD

**Issue by pawchen** [dotnet/roslyn#65482]
**Version Used**:  VS17.4

**Steps to Reproduce**:

1. Define some struct
```C#
struct A
{
  public required int F1 { get; init; }
  public int F2 { get; init; }
}
```
2. Initialize it somewhere, type `new A Enter {`, `$$` means my caret:
```C#
new A
{ $$}
```
3. `F1` is aggressively selected at this point, but muscle memory leads me to hit Enter again, expecting
```C#
new A
{
    $$
}
```
Actually
```C#
new A
{ F1$$}
```

**Comment by CyrusNajmabadi** [comment]
Yes.  We should not be hard selecting items here.  If we're doing that, that's def a bug.

**Comment by pawchen** [comment]
It appears that it has something to do with the `required` keyword. This only happens when the struct has a required member.
